workflow/scripts/build_existing_chp_de.py

```diff
@@ -146,10 +181,14 @@ def BP(cap,year):
         path = "../submodules/pypsa-eur/scripts"
         sys.path.insert(0, os.path.abspath(path))
         from _helpers import mock_snakemake
+
         snakemake = mock_snakemake("build_existing_chp_de")
 
     logging.basicConfig(level=snakemake.config["logging"]["level"])
 
+    options = snakemake.params.sector
+    opts = options[0].split("-")
```

Comment:
```python
opts = snakemake.wildcards.sector_opts.split("-")
``` 
Taking the first entry from the config may not be consistent with the wildcards.

Reply:
On second thought, I don't think the model has access to the `sector_opts` in this rule yet. Maybe carry the information, whether it's industry CHP or not, over to the subsequent rules where you'd then do the filtering.